runtime/build.gradle

```diff
@@ -29,6 +29,7 @@ def deployMode = property('native.deploy') == 'true'
 
 kotlin {
     infra {
+        target('wasm32')
```

Comment:
It sufficient to include only this line, without the rest; `infra` block will automatically handle source sets.

Please put it under `if (deployMode)`. See, for regular development (when you write code, then run tests) for a faster feedback loop better to include only one target – host platform (either linux or macos). Other targets (Windows, ios, webasm) are included under `deployMode` so they're tested and built only on CI in order not to waste developer's time.